AirCasting/SessionViews/ShareViewModal.swift

```diff
@@ -0,0 +1,72 @@
+// Created by Lunar on 28/06/2021.
+//
+
+import AirCastingStyling
+import SwiftUI
+
+struct ShareViewModal: View {
+    @Environment(\.presentationMode) var presentationMode
+    @State var email: String = ""
+    
+    var body: some View {
+        VStack(alignment: .leading, spacing: 20) {
+            VStack(alignment: .leading, spacing: 5) {
+                title
+                description
+            }
+            checkBox
+            shareButton
+            descriptionMail
+            createTextfield(placeholder: "Email", binding: $email)
+                .padding(.vertical)
+            oKButton
+            cancelButton
+        }
+        .padding()
+    }
+    
+    private var title: some View {
+        Text(Strings.sessionShare.title)
+            .font(.title)
```

Comment:
Are you sure it's the right font? :) 
We don't use system fonts. 
I see two possibilities here --> either use our Font.muli/Font.moderat or create an extension for this and set the right font for .title, .subheadline, etc :) 
The second solution has more advantages - it supports some dynamic text scaling. 
But it's your call :)

Reply:
Sorry for that, I've changed it to another. IDK why it left by itself from my commit